pkg/cli/bootstrap.go

```diff
@@ -104,6 +106,36 @@ func urfaveRegisterShutdownHandler(cancel context.CancelFunc) {
 	}()
 }
 
+func addCommonProps(ctx context.Context, c telefork.Client) {
```

Comment:
This function does 3 things, builds the common props, adds it to trace and adds it to the client. Should it just do one thing instead, build the CommonProps?